conf/messages

```diff
@@ -37,52 +37,36 @@ unit-input.assistive.percent=do not enter the percent sign
-landingPage.list1.bullet5=create a Capital Gains Tax on UK property account
-landingPage.list1.bullet6=view and amend your previous and current returns
-landingPage.h2.2=If you’re a non-resident
-landingPage.p2=You can use this service to report the disposal and pay the tax on:
-landingPage.list2.bullet1=the sale or disposal of residential and non-residential UK property or land
-landingPage.list2.bullet2=indirect disposals
-landingPage.p3=You must report the disposal by the 30-day deadline, even if:
-landingPage.list3.bullet1=you’ve no tax to pay
-landingPage.list3.bullet2=you’ve made a loss
-landingPage.signIn=Sign in
-landingPage.signIn.p1=<a href="{0}">Sign in</a> if you already have an account.
-landingPage.beforeYouStart=Before you start
-landingPage.beforeYouStart.p1=You’ll need a <a href="{0}">Government Gateway</a> user ID and password to use this service. If you do not have a user ID, you can create one.
-landingPage.beforeYouStart.p2=You can <a href="{0}">contact HMRC</a> if you need help accessing the service.
-landingPage.whatYoullNeed=What you’ll need to complete a return
-landingPage.whatYoullNeed.p1=You’ll need to provide the:
-landingPage.whatYoullNeed.bullet1=property’s address and postcode
-landingPage.whatYoullNeed.bullet2=date you got the property
-landingPage.whatYoullNeed.bullet3=date you exchanged contracts when you were selling or disposing of the property
-landingPage.whatYoullNeed.bullet4=date you stopped being the property’s owner (completion date)
-landingPage.whatYoullNeed.bullet5=property’s value when you got it, and when you disposed of it
-landingPage.whatYoullNeed.bullet6=costs of buying, selling or making improvements to the property
-landingPage.whatYoullNeed.bullet7=details of any tax reliefs, allowances or exemptions you’re entitled to claim
-landingPage.whatYoullNeed.bullet8=property type, if you’re a non-resident
-landingPage.ifYoureAnAgent=If you’re an agent
-landingPage.ifYoureAnAgent.p1=Go to your <a href="{0}">agent services account</a> to access this service. If you do not already have one, you can <a href="{1}">apply for an agent services account</a>.
-landingPage.ifYoureRepresentingATrust=If you’re representing a trust
-landingPage.ifYoureRepresentingATrust.p1=The trust will need to be <a href="{0}">registered</a>. You will need the trust’s Self Assessment Unique Taxpayer Reference.
+landingPage.intro.p1=You may have to pay <a href="{0}">Capital Gains Tax</a> if you make a profit (‘gain’) when you sell (or ‘<a href="{0}">dispose of</a>’) property that’s not your home. For example, buy-to-let properties, land or inherited property.
+landingPage.intro.p2=In most cases you do not need to pay the tax when you <a href="{0}">sell your main home</a>.
+landingPage.createAccount.h2=Create a Capital Gains Tax on UK property account
+landingPage.createAccount.p1=Before 6 April 2020, you'll be able sign in to this service using your Government Gateway user ID and password to:
```

Comment:
this looks like a straight quote in `you'll`

Reply:
yep replaced